src/app/ui/dynamics_popup.cpp

```diff
@@ -289,15 +306,20 @@ void DynamicsPopup::setOptionsGridVisibility(bool state)
     expandWindow(sizeHint());
 }
 
-int DynamicsPopup::ditheringIndex() const {
+std::string DynamicsPopup::ditheringMatrixName() const {
   if (m_ditheringSel)
-    return m_ditheringSel->getSelectedItemIndex();
-  return 0;
+    return m_ditheringSel->getItemText(
+      m_ditheringSel->getSelectedItemIndex());
+  return Strings::dithering_selector_no_dithering();
```

Comment:
Not sure if this should return an empty string in this case:
```suggestion
  return std::string();
```